test/ec2-metadata-test-proxy/cmd/ec2-metadata-test-proxy.go

```diff
@@ -51,6 +51,7 @@ const (
 )
 
 var startTime int64 = time.Now().Unix()
+var spotInterruptionTime string = time.Now().Local().Add(time.Minute * time.Duration(2)).Format(time.RFC3339)
```

Comment:
should use `time.Now().UTC()...` rather than `Local()`

Reply:
I realize now that I wrote that original line which you just moved up.... how embarrassing. But it should indeed be UTC() rather than Local()